extensions/skyportal/static/js/components/AlertsSearchButton.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import { Link } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+
+import * as alertsActions from "../ducks/alerts";
+
+const AlertsSearchButton = ({ objID, ra, dec, radius = 3 }) => {
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    dispatch(alertsActions.fetchAlerts({ object_id: objID, ra, dec, radius }));
```

Comment:
There may be sources with other `objectId`'s within 3 arcseconds - let's not restrict the search to the current `objID` only:

```suggestion
    dispatch(alertsActions.fetchAlerts({ ra, dec, radius }));
```